tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -519,3 +519,34 @@ def test_cuDoubleComplex(self):
         ker = mod.get_function('test_downcast')
         ker((grid,), (block,), (a, out_down))
         assert (out_down == a.astype(cupy.complex64)).all()
+
+
+class TestRawGridSync(unittest.TestCase):
+
+    def setUp(self):
+        global _test_cache_dir
+        _test_cache_dir = tempfile.mkdtemp()
+        os.environ['CUPY_CACHE_DIR'] = _test_cache_dir
```

Comment:
Please write the corresponding `tearDown` logic.

Reply:
Fixed.